homeassistant/components/elmax/translations/en.json

```diff
@@ -1,14 +1,14 @@
 {
     "config": {
         "abort": {
-            "already_configured": "Device is already configured"
+            "single_instance_allowed": "There already is an integration for that Elmaxc panel."
         },
         "error": {
             "bad_auth": "Invalid authentication",
             "invalid_pin": "The provided pin is invalid",
             "network_error": "A network error occurred",
             "no_panel_online": "No online Elmax control panel was found.",
-            "unknown_error": "An unexpected error occurred"
```

Comment:
We still need to include the `"unknown"` error key here with a reference to the common string as value.

Reply:
Done.